### ISSUE TITLE
Z-fighting

### ISSUE THREAD

**Issue by Blue-Beaker** [shedaniel/CustomSelectionBox-New#8]
![2019-03-17_03 34 53](https://user-images.githubusercontent.com/17115547/54481008-469df100-486a-11e9-9c27-017bdeb13579.png)
![2019-03-17_03 35 05](https://user-images.githubusercontent.com/17115547/54481009-469df100-486a-11e9-9920-7567ba35ef1c.png)
Z-fighting when disable depth off.
I think if the translucent section of the selection box is slightly larger than MC default, the Z-fighting will not happen.So I suggest adding an option to adjust selection box size, so that users can adjust it to make best effects for their devices.

**Comment by shedaniel** [comment]
Hmm, I should've expended the selection box by a little bit, not sure if I removed that

**Comment by Mhowser** [comment]
there is still some Z-fighting for the latest release.

**Comment by shedaniel** [comment]
On the 1.15 version?

**Comment by Mhowser** [comment]
Yes, the one you uploaded recently, if you look at the vanilla outline very closely, it is slightly bigger than the block. The mod's outline is part of the block edges, which causes the z fighting
